src/util/logger.js

```diff
@@ -0,0 +1,24 @@
+// Use a custom logger application
+const LOGGER_HOST = process.env?.REACT_APP_LOGGER_HOST || "http://localhost"
```

Comment:
Since I have little knowledge about `https`, to support `https` eventually should work be done in the logger application only, and the client here is good to just to change https URL in `env`? Or should we do something additionally in the dashboard to support `https`?

Reply:
If nothing to be done for `https` for now or you prefer to add it later in a separate PR, this is a good start and I am good to approve and merge.

Reply:
I don't think any real changes need to be made to the Dashboard to support sending data via HTTPS, since it already supports running with HTTPS.